.github/workflows/build.yml

```diff
@@ -42,7 +42,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        os: [ubuntu-18.04, macOS-10.14]
+        os: [ubuntu-18.04]
```

Comment:
remove `matrix` section and change runs-on to ubuntu-18.04 e.g `runs-on: ubuntu-18.04`

Reply:
done